packages/features/bookings/Booker/config.ts

```diff
@@ -70,7 +70,7 @@ export const resizeAnimationConfig: ResizeAnimationConfig = {
       "meta main main"
       `,
       gridTemplateColumns: "var(--booker-meta-width) var(--booker-main-width)",
-      gridTemplateRows: "auto",
+      gridTemplateRows: "1fr 0fr",
```

Comment:
This fixes the safari issue. 0fr for second row, because the 2 rows show the exact same template areas. So content will still be visible. Having 2 rows is needed (even though they are the same), to make the animations animate to the proper places. They would space out if suddenly a second row got added.